azext_iot/tests/digitaltwins/test_dt_resource_lifecycle_int.py

```diff
@@ -343,6 +341,26 @@ def test_dt_endpoints_routes(self):
                 EP_RG,
             )
         ).as_json()["id"]
+
+        self._create_storage_account()
```

Comment:
This explained why i started to see many SA popping up in the s360 report.
Do have two questions here (not blocking)
Are we delete them in the tear down?
Can we create them with key access disabled so they won't be captured on the s360 board?
![image](https://github.com/user-attachments/assets/58f4fad5-cb70-4445-a264-87106e88afa5)

Reply:
Side note, i was thinking creating an azure function to disable key access for all storage accounts 24h before the s360 meeting, but will i break the any tests if we do need to use the key access?

Reply:
Yup, these are deleted in the teardown and we have a backup task to clean them up nightly.

Regarding that function, it would certainly break tests that run at that time if they're relying on connection strings for endpoints or other interactions - which we do heavily in hub since it's a common customer scenario still